foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/server/TcpServer.java

```diff
@@ -18,23 +18,35 @@
 package org.apache.servicecomb.foundation.vertx.server;
 
 import java.net.InetSocketAddress;
+import java.util.concurrent.atomic.AtomicInteger;
 
+import org.apache.servicecomb.foundation.common.event.EventManager;
 import org.apache.servicecomb.foundation.common.net.URIEndpointObject;
 import org.apache.servicecomb.foundation.ssl.SSLCustom;
 import org.apache.servicecomb.foundation.ssl.SSLOption;
 import org.apache.servicecomb.foundation.ssl.SSLOptionFactory;
 import org.apache.servicecomb.foundation.vertx.AsyncResultCallback;
+import org.apache.servicecomb.foundation.vertx.ClientConnectedEvent;
 import org.apache.servicecomb.foundation.vertx.VertxTLSBuilder;
 
+import com.netflix.config.DynamicPropertyFactory;
+
 import io.vertx.core.Vertx;
 import io.vertx.core.net.NetServer;
 import io.vertx.core.net.NetServerOptions;
 
 public class TcpServer {
   private URIEndpointObject endpointObject;
 
+  private final AtomicInteger connectedCounter;
```

Comment:
TcpServer  will be created for each HighwayServerVerticle instance
so the counter number in server is not correct.

Reply:
your means I need move it to HighwayServerVerticle instance?

Reply:
I think we only deploy one HighwayServerVerticle instance (as a spring bean), so may not problem ?

Reply:
if serivcecomb.highway.thread-count is n, then HighwayServerVerticle  instance count is n

Reply:
sorry, forgotten

Reply:
Done